docs/source/tracking.rst

```diff
@@ -750,6 +833,18 @@ location to server's artifact store. This will be used as artifact location for
 experiments that do not specify one. Once you create an experiment, ``--default-artifact-root``
 is no longer relevant to that experiment.
 
+Starting a server with the ``--serve-artifacts`` option defined will enable proxy access for artifacts.
+The uri ``mlflow-artifacts:/`` will serve in place of the ``--default-artifact-root`` path configured during
+server start to refer to artifact storage locations.
+
+.. important::
+    When operating an MLflow tracking server with ``--serve-artifacts`` option enabled, the parameter
+    ``--default-artifact-root`` will be set to ``mlflow-artifacts:/`` as a proxy root to the file store location
+    for artifact storage. Artifact handling can be accomplished through the use of this proxy uri. Access credentials
```

Comment:
These first two sentences are redundant given the information above, right?

Reply:
definitely. removed them.